clients/fides-js/rollup.config.mjs

```diff
@@ -152,6 +152,9 @@ SCRIPTS.forEach(({ name, gzipErrorSizeKb, gzipWarnSizeKb, isExtension }) => {
         name: isExtension ? undefined : "Fides",
         format: isExtension ? undefined : "umd",
```

Comment:
nit:
```suggestion
        format: isExtension ? "es" : "umd",
```

This just feels like it's better at communicating what we're doing here